suite-native/app/src/App.tsx

```diff
@@ -128,4 +129,4 @@ const PureApp = () => (
     </GestureHandlerRootView>
 );
 
-export const App = Sentry.wrap(PureApp);
+export const App = isDebugEnv() ? PureApp : Sentry.wrap(PureApp);
```

Comment:
I not sure if this will solve issue, in other envs than debug it will use Sentry.wrap even if Sentry is disabled. Maybe that is not issue because it won't do anything if Sentry is disabled but I am not sure.

Reply:
I agree. I didnt realise init is called in useEffect. But I would say it will work the same as it worked until now and the warning wont bother us while developing: https://github.com/trezor/trezor-suite/pull/10555#discussion_r1446273799

Reply:
Ok, seems reasonable.